pkg/controller/backup_storage_location_controller.go

```diff
@@ -52,7 +52,7 @@ type BackupStorageLocationReconciler struct {
 
 // +kubebuilder:rbac:groups=velero.io,resources=backupstoragelocations,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=velero.io,resources=backupstoragelocations/status,verbs=get;update;patch
-func (r *BackupStorageLocationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+func (r *BackupStorageLocationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
```

Comment:
We can address this in a follow up PR, but I think that now that `Reconcile` takes a context, we can upgrade our existing controllers to not have their [own context](https://github.com/vmware-tanzu/velero/blob/main/pkg/controller/backup_storage_location_controller.go#L41) but instead use this passed in context. Looking at the code from controller-runtime, this is the context that is now passed in to `s.mgr.Start` in `server.go`. I'll open an issue to address this.

Reply:
Created https://github.com/vmware-tanzu/velero/issues/3353